Adafruit_IO/errors.py

```diff
@@ -20,8 +20,11 @@
 # SOFTWARE.
 
 import json, requests
+from paho.mqtt.client import error_string
 
-# MQTT RC Error Types
+# MQTT RC Error Types *** OBSOLETE *** 
+# See error_string() in client.py and enums.py from paho instead
+# https://github.com/eclipse/paho.mqtt.python/blob/4eeb431f5ae72b42474cec42641fca1daa91c4b0/src/paho/mqtt/client.py#L291-L404
 MQTT_ERRORS   = [ 'Connection successful',
```

Comment:
If we aren't using `MQTT_ERRORS`, can we remove?

Reply:
Was going to leave it for one fixed version, in case people are importing it for their own "reasons", but it maybe more sensible to remove and in that way let them know it's no longer available/useful.

Reply:
I think we should remove it if it's not used.